### PR TITLE
Devop: Kleros name tags integration for address book

### PR DESCRIPTION
**Pull request by Abyscuit**
### Devop

* \[x] Added entry to ./changelog
* \[x] Add PR label

### CODE REVIEW

**Review by gamalielhere — APPROVED** [review]

File: src/modules/address-book/ModuleAddressBook.vue [hunk 1]
```diff
@@ -201,7 +201,17 @@ export default {
             const reverseName = await this.nameResolver.resolveAddress(
               this.inputAddr
             );
-            this.resolvedAddr = reverseName.name ? reverseName.name : '';
+            if (!reverseName.name) {
+              reverseName.name =
+                (
+                  await getAddressInfo(
+                    toChecksumAddress(this.inputAddr),
+                    'https://ipfs.kleros.io'
```
Comment (gamalielhere) [inline src/modules/address-book/ModuleAddressBook.vue:209]: This doesn't need to be in connections?

**Review by gamalielhere — CHANGES_REQUESTED** [review]

File: src/modules/address-book/ModuleAddressBook.vue (hunk 1 above)
Comment (Abyscuit) [reply src/modules/address-book/ModuleAddressBook.vue:209]: It seemed to work fine without it but I will add to connections

**Review by gamalielhere — APPROVED** [review]

**Review by andrewdahn — APPROVED** [review]

File: src/modules/address-book/ModuleAddressBook.vue [hunk 2]
```diff
@@ -2,7 +2,7 @@
   <div>
     <mew-address-select
       ref="addressSelect"
-      :resolved-addr="resolvedAddr"
+      :hint="resolvedAddr"
```
Comment (gamalielhere) [inline src/modules/address-book/ModuleAddressBook.vue:5]: don't you still need resolved-addr here? hint should just be for kleros and ens reverse name. resolve-addr is for ens names

**Review by gamalielhere — CHANGES_REQUESTED** [review]

**Review by gamalielhere — APPROVED** [review]

File: src/modules/address-book/components/AddressBookAddEdit.vue [hunk 3]
```diff
@@ -158,15 +160,15 @@ export default {
           this.$t('interface.address-book.validations.nickname-length')
       ];
     },
-    actualValidAddress() {
-      if (!this.validAddress) return '';
-      return isAddress(this.lowercaseAddressToAdd)
-        ? this.lowercaseAddressToAdd
-        : this.addressToAdd;
-    },
+    // actualValidAddress() {
```
Comment (gamalielhere) [inline src/modules/address-book/components/AddressBookAddEdit.vue:163]: Remove commented code

**Review by gamalielhere — CHANGES_REQUESTED** [review]

**Pull request merged**